src/components/templates/PartyMap/PartyMapAdmin.tsx

```diff
@@ -1,41 +1,69 @@
-import React, { useState, useEffect, FC, useCallback } from "react";
-import { useSelector } from "hooks/useSelector";
+import React, { useState, useEffect, useCallback } from "react";
+
+import { RootState } from "index";
 import firebase from "firebase/app";
+
 import { PartyMapVenue } from "types/PartyMapVenue";
-import { RootState } from "index";
 
+import { useSelector } from "hooks/useSelector";
+
+// TODO: we should move this to a common helper location
+const makeUpdateBanner = (
+  venueId: string,
+  onError: (errorMsg: string) => void
+) => (message?: string) => {
+  const params = {
+    venueId,
+    bannerMessage: message ? message : "",
+  };
+
+  firebase
+    .functions()
+    .httpsCallable("venue-adminUpdateBannerMessage")(params)
+    .catch((e) => onError(e.toString()));
```

Comment:
I think we can get rid of this function and just use simple doc.update

Reply:
[OOS/future]

Reply:
extracted this to `api/partyMapAdmin` in https://github.com/sparkletown/sparkle/pull/840/commits/eef8c8a2175f61f9bbe7c9dc4e741763a4aa0d61